tests/network/vmi_multus.go

```diff
@@ -929,8 +929,11 @@ var _ = Describe("[Serial]SRIOV", func() {
 			vmi1.Spec.Domain.Devices.Interfaces[1].MacAddress = mac1.String()
 			vmi2.Spec.Domain.Devices.Interfaces[1].MacAddress = mac2.String()
 
-			vmi1 = startVmi(vmi1)
-			vmi2 = startVmi(vmi2)
+			sriovNodes := getNodesWithAllocatedResource(virtClient, sriovResourceName)
+			Expect(len(sriovNodes)).To(BeNumerically(">", 0))
+			vmi1 = tests.StartVmOnNode(vmi1, sriovNodes[0].Name)
+			vmi2 = tests.StartVmOnNode(vmi2, sriovNodes[0].Name)
```

Comment:
nit: I'd add a local variable named and re-use it here.

Reply:
Done